test/e2e/node/pods.go

```diff
@@ -55,9 +55,9 @@ var _ = SIGDescribe("Pods Extended", func() {
 		/*
 			Release : v1.15
 			Testname: Pods, delete grace period
-			Description: Create a pod, make sure it is running. Using the http client send a 'delete' with gracePeriodSeconds=30. Pod SHOULD get deleted within 30 seconds.
+			Description: Create a pod, make sure it is running. Using the http client send a 'delete' with gracePeriodSeconds=30. Pod SHOULD get terminated within gracePeriodSeconds and removed from API server within a window.
```

Comment:
Nit: Worth adding a ballpark range for the window value? I.e. is it minutes, 10s of minutes, etc...

Reply:
the window is not specific right now, 30s is too tight.  2x30s was still not perfect based on @liggitt past attempts, but trying 3x30s to see if we can still ensure the function works while we work out more precise SLA for kubelet deleting the pod from the API server.

Reply:
Gotcha! Could be helpful to have that info in the commit message or as a comment so someone looking at this in the future doesn't need to return to the pr description.